tests/pipelines/test_pipelines_automatic_speech_recognition.py

```diff
@@ -123,7 +123,7 @@ def run_pipeline_test(self, speech_recognizer, examples):
             outputs = speech_recognizer(audio, return_timestamps=True)
             self.assertIsInstance(outputs["chunks"], list)
             nb_chunks = len(outputs["chunks"])
-            self.assertGreaterThan(nb_chunks, 0)
+            self.assertTrue(nb_chunks > 0)
```

Comment:
Oups this should not be modified either!

Reply:
When running it initially I got an error that this function is not defined and I didn't find its occurrence anywhere in the codebase either?

Reply:
Should be `self.assertGreater` here ([doc](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertGreater)).

Reply:
Makes sense, thanks!